weaver/wps_restapi/swagger_definitions.py

```diff
@@ -3613,6 +3615,56 @@ class RequirementClass(ExtendedSchemaNode):
     description = "CWL requirement class specification."
 
 
+class CudaRequirementSpecification(PermissiveMappingSchema):
+    cudaVersionMin = ExtendedSchemaNode(
+        String(),
+        example="11.4",
+        title="Cuda version minimum",
+        description="The minimum Cuda version required."
+    )
+    cudaComputeCapability = ExtendedSchemaNode(
+        String(),
+        example="3.0",
+        title="Cuda compute capability",
+        description="The compute capability supported by the GPU."
+    )
```

Comment:
For each of the "version" fields, use the `SemanticVersion` validator with the appropriate pattern.